docs/governance/canonical-reconciliation-register.md

```
@@ -263,6 +268,8 @@ ADR-0007, ADR-0020, ADR-0029, ADR-0021, ADR-0022, ADR-0023, ADR-0024,
 ADR-0025, and ADR-0026 respectively.
 #11 is resolved by ADR-0034 and the reconciled authorization/API/history/
 collaboration specifications without authorizing production implementation.
+#50 is resolved by ADR-0035 and the reconciled collaboration specification/
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)